doc/source/whatsnew/v1.2.0.rst

```diff
@@ -13,6 +13,25 @@ including other versions of pandas.
 Enhancements
 ~~~~~~~~~~~~
 
+.. _whatsnew_120.binary_handle_to_csv:
+
+Support for binary file handles in ``to_csv``
```

Comment:
can you add this example to the io.rst section as well

Reply:
I added a new subsection in "Data Handling". Is that an appropriate place?